### ISSUE TITLE
BenchmarkDotNet.exe crash doesn't release resources

### ISSUE THREAD

**Issue by Jason1269** [dotnet/BenchmarkDotNet#1639]
Benchmarkdotnet fails to release handles when it desn't exit cleanly.  On one instance I accidentally removed some artifacts (git clean -fdx), causing app to crash.  After this, I wasn't able delete files nor I rebuild my project

> C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4571,5): warning MSB3026: Could not copy "C:\Source\proj\mydll.dll" to "bin\Test\net5.0\mydll.dll". Beginning retry 10 in 1000ms. The process cannot access the file 'bin\Test\net5.0\mydll.dll' because it is being used by another process. The file is locked by: "Benchmark.exe (16788)"

Handle.exe doesn't show any open handles either, and neither Benchmark.exe nor pid 16788 show in task manager

> handle mydll.dll
>
> Nthandle v4.22 - Handle viewer
> Copyright (C) 1997-2019 Mark Russinovich
> Sysinternals - www.sysinternals.com
> 
> No matching handles found.

It does show up when using pslist

> pslist 16788
> 
> PsList v1.4 - Process information lister
> Copyright (C) 2000-2016 Mark Russinovich
> Sysinternals - www.sysinternals.com
> 
> Process information for MyPC:
> 
> Name                Pid Pri Thd  Hnd   Priv        CPU Time    Elapsed Time
> Benchmark         16788   8   0    0   1204     0:00:00.000     0:30:27.830

I have full admin privilege on my pc and am running task manager, handle, and pslist in elevated mode

**Comment by adamsitnik** [comment]
Hi @Jason1269 

Thank you for a detailed bug report. Is the bug reproducible for you?

I've tried to run `git clean -xfd` while running `dotnet run -c Release -f netcoreapp2.1 --filter BenchmarkDotNet.Samples.IntroArgumentsSource.SingleArgument` in our samples project and I got an exception at the end, but overall it exited and released all resources:

<details>

```log
C:\Projects\BenchmarkDotNet\samples\BenchmarkDotNet.Samples> dotnet run -c Release -f netcoreapp2.1 --filter BenchmarkDotNet.Samples.IntroArgumentsSource.SingleArgument                                                                                                   C:\Program Files\dotnet\sdk\5.0.102\Microsoft.Common.CurrentVersion.targets(1662,5): warning NU1702: ProjectReference 'C:\Projects\BenchmarkDotNet\src\BenchmarkDotNet.Disassembler.x64\BenchmarkDotNet.Disassembler.x64.csproj' was resolved using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETStandard,Version=v2.0'. This project may not be fully compatible with your project. [C:\Projects\BenchmarkDotNet\src\BenchmarkDotNet\BenchmarkDotNet.csproj]
C:\Program Files\dotnet\sdk\5.0.102\Microsoft.Common.CurrentVersion.targets(1662,5): warning NU1702: ProjectReference 'C:\Projects\BenchmarkDotNet\src\BenchmarkDotNet.Disassembler.x86\BenchmarkDotNet.Disassembler.x86.csproj' was resolved using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETStandard,Version=v2.0'. This project may not be fully compatible with your project. [C:\Projects\BenchmarkDotNet\src\BenchmarkDotNet\BenchmarkDotNet.csproj]
// Validating benchmarks:
// ***** BenchmarkRunner: Start   *****
// ***** Found 2 benchmark(s) in total *****
// ***** Building 1 exe(s) in Parallel: Start   *****
// start dotnet restore  /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true in C:\Projects\BenchmarkDotNet\samples\BenchmarkDotNet.Samples\bin\Release\netcoreapp2.1\59fc3eed-cfb3-4860-9ce5-5cbbde129a14
// command took 1.62s and exited with 0
// start dotnet build -c Release  --no-restore /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true in C:\Projects\BenchmarkDotNet\samples\BenchmarkDotNet.Samples\bin\Release\netcoreapp2.1\59fc3eed-cfb3-4860-9ce5-5cbbde129a14
// command took 2.77s and exited with 0
// ***** Done, took 00:00:04 (4.6 sec)   *****
// Found 2 benchmarks:
//   IntroArgumentsSource.SingleArgument: DefaultJob [time=00:00:00.0100000]
//   IntroArgumentsSource.SingleArgument: DefaultJob [time=00:00:00.1000000]

// **************************
// Benchmark: IntroArgumentsSource.SingleArgument: DefaultJob [time=00:00:00.0100000]
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "59fc3eed-cfb3-4860-9ce5-5cbbde129a14.dll" --benchmarkName "BenchmarkDotNet.Samples.IntroArgumentsSource.SingleArgument(time: 00:00:00.0100000)" --job "Default" --benchmarkId 0 in C:\Projects\BenchmarkDotNet\samples\BenchmarkDotNet.Samples\bin\Release\netcoreapp2.1\59fc3eed-cfb3-4860-9ce5-5cbbde129a14\bin\Release\netcoreapp2.1
// BeforeAnythingElse

// Benchmark Process Environment Information:
// Runtime=.NET Core 2.1.24 (CoreCLR 4.6.29518.01, CoreFX 4.6.29518.01), X64 RyuJIT
// GC=Concurrent Workstation
// Job: DefaultJob

OverheadJitting  1: 1 op, 286800.00 ns, 286.8000 us/op
WorkloadJitting  1: 1 op, 13704900.00 ns, 13.7049 ms/op

OverheadJitting  2: 16 op, 275500.00 ns, 17.2188 us/op
WorkloadJitting  2: 16 op, 266941700.00 ns, 16.6839 ms/op

WorkloadPilot    1: 16 op, 259584700.00 ns, 16.2240 ms/op
WorkloadPilot    2: 32 op, 538759600.00 ns, 16.8362 ms/op

OverheadWarmup   1: 32 op, 2100.00 ns, 65.6250 ns/op
OverheadWarmup   2: 32 op, 300.00 ns, 9.3750 ns/op
OverheadWarmup   3: 32 op, 300.00 ns, 9.3750 ns/op
OverheadWarmup   4: 32 op, 200.00 ns, 6.2500 ns/op
OverheadWarmup   5: 32 op, 400.00 ns, 12.5000 ns/op
OverheadWarmup   6: 32 op, 300.00 ns, 9.3750 ns/op

OverheadActual   1: 32 op, 400.00 ns, 12.5000 ns/op
OverheadActual   2: 32 op, 400.00 ns, 12.5000 ns/op
OverheadActual   3: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual   4: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual   5: 32 op, 400.00 ns, 12.5000 ns/op
OverheadActual   6: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual   7: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual   8: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual   9: 32 op, 200.00 ns, 6.2500 ns/op
OverheadActual  10: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual  11: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual  12: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual  13: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual  14: 32 op, 400.00 ns, 12.5000 ns/op
OverheadActual  15: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual  16: 32 op, 200.00 ns, 6.2500 ns/op
OverheadActual  17: 32 op, 200.00 ns, 6.2500 ns/op
OverheadActual  18: 32 op, 300.00 ns, 9.3750 ns/op
OverheadActual  19: 32 op, 100.00 ns, 3.1250 ns/op
OverheadActual  20: 32 op, 300.00 ns, 9.3750 ns/op

WorkloadWarmup   1: 32 op, 532563300.00 ns, 16.6426 ms/op
WorkloadWarmup   2: 32 op, 538771500.00 ns, 16.8366 ms/op
WorkloadWarmup   3: 32 op, 532324600.00 ns, 16.6351 ms/op
WorkloadWarmup   4: 32 op, 532247500.00 ns, 16.6327 ms/op
WorkloadWarmup   5: 32 op, 539084700.00 ns, 16.8464 ms/op
WorkloadWarmup   6: 32 op, 531935800.00 ns, 16.6230 ms/op

// BeforeActualRun
WorkloadActual   1: 32 op, 540864700.00 ns, 16.9020 ms/op
WorkloadActual   2: 32 op, 538683200.00 ns, 16.8339 ms/op
WorkloadActual   3: 32 op, 532436200.00 ns, 16.6386 ms/op
WorkloadActual   4: 32 op, 532141600.00 ns, 16.6294 ms/op
WorkloadActual   5: 32 op, 539217900.00 ns, 16.8506 ms/op
WorkloadActual   6: 32 op, 532421300.00 ns, 16.6382 ms/op
WorkloadActual   7: 32 op, 532484400.00 ns, 16.6401 ms/op
WorkloadActual   8: 32 op, 538916800.00 ns, 16.8412 ms/op
WorkloadActual   9: 32 op, 532395700.00 ns, 16.6374 ms/op
WorkloadActual  10: 32 op, 532092100.00 ns, 16.6279 ms/op
WorkloadActual  11: 32 op, 538956900.00 ns, 16.8424 ms/op
WorkloadActual  12: 32 op, 532279500.00 ns, 16.6337 ms/op
WorkloadActual  13: 32 op, 532389400.00 ns, 16.6372 ms/op
WorkloadActual  14: 32 op, 538389800.00 ns, 16.8247 ms/op
WorkloadActual  15: 32 op, 532398200.00 ns, 16.6374 ms/op

// AfterActualRun
WorkloadResult   1: 32 op, 540864400.00 ns, 16.9020 ms/op
WorkloadResult   2: 32 op, 538682900.00 ns, 16.8338 ms/op
WorkloadResult   3: 32 op, 532435900.00 ns, 16.6386 ms/op
WorkloadResult   4: 32 op, 532141300.00 ns, 16.6294 ms/op
WorkloadResult   5: 32 op, 539217600.00 ns, 16.8506 ms/op
WorkloadResult   6: 32 op, 532421000.00 ns, 16.6382 ms/op
WorkloadResult   7: 32 op, 532484100.00 ns, 16.6401 ms/op
WorkloadResult   8: 32 op, 538916500.00 ns, 16.8411 ms/op
WorkloadResult   9: 32 op, 532395400.00 ns, 16.6374 ms/op
WorkloadResult  10: 32 op, 532091800.00 ns, 16.6279 ms/op
WorkloadResult  11: 32 op, 538956600.00 ns, 16.8424 ms/op
WorkloadResult  12: 32 op, 532279200.00 ns, 16.6337 ms/op
WorkloadResult  13: 32 op, 532389100.00 ns, 16.6372 ms/op
WorkloadResult  14: 32 op, 538389500.00 ns, 16.8247 ms/op
WorkloadResult  15: 32 op, 532397900.00 ns, 16.6374 ms/op

// AfterAll
// Benchmark Process 8636 has exited with code 0

Mean = 16.721 ms, StdErr = 0.028 ms (0.17%), N = 15, StdDev = 0.110 ms
Min = 16.628 ms, Q1 = 16.637 ms, Median = 16.639 ms, Q3 = 16.837 ms, Max = 16.902 ms
IQR = 0.200 ms, LowerFence = 16.337 ms, UpperFence = 17.138 ms
ConfidenceInterval = [16.604 ms; 16.838 ms] (CI 99.9%), Margin = 0.117 ms (0.70% of Mean)
Skewness = 0.43, Kurtosis = 1.18, MValue = 2

// **************************
// Benchmark: IntroArgumentsSource.SingleArgument: DefaultJob [time=00:00:00.1000000]
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "59fc3eed-cfb3-4860-9ce5-5cbbde129a14.dll" --benchmarkName "BenchmarkDotNet.Samples.IntroArgumentsSource.SingleArgument(time: 00:00:00.1000000)" --job "Default" --benchmarkId 1 in C:\Projects\BenchmarkDotNet\samples\BenchmarkDotNet.Samples\bin\Release\netcoreapp2.1\59fc3eed-cfb3-4860-9ce5-5cbbde129a14\bin\Release\netcoreapp2.1
// BeforeAnythingElse

// Benchmark Process Environment Information:
// Runtime=.NET Core 2.1.24 (CoreCLR 4.6.29518.01, CoreFX 4.6.29518.01), X64 RyuJIT
// GC=Concurrent Workstation
// Job: DefaultJob

OverheadJitting  1: 1 op, 268400.00 ns, 268.4000 us/op
WorkloadJitting  1: 1 op, 106896200.00 ns, 106.8962 ms/op

WorkloadPilot    1: 2 op, 211037400.00 ns, 105.5187 ms/op
WorkloadPilot    2: 3 op, 333608200.00 ns, 111.2027 ms/op
WorkloadPilot    3: 4 op, 404011500.00 ns, 101.0029 ms/op
WorkloadPilot    4: 5 op, 516716600.00 ns, 103.3433 ms/op

WorkloadWarmup   1: 5 op, 513077400.00 ns, 102.6155 ms/op
WorkloadWarmup   2: 5 op, 515947500.00 ns, 103.1895 ms/op
WorkloadWarmup   3: 5 op, 516680300.00 ns, 103.3361 ms/op
WorkloadWarmup   4: 5 op, 517984200.00 ns, 103.5968 ms/op
WorkloadWarmup   5: 5 op, 515071300.00 ns, 103.0143 ms/op
WorkloadWarmup   6: 5 op, 523182200.00 ns, 104.6364 ms/op
WorkloadWarmup   7: 5 op, 516182400.00 ns, 103.2365 ms/op

// BeforeActualRun
WorkloadActual   1: 5 op, 514654500.00 ns, 102.9309 ms/op
WorkloadActual   2: 5 op, 528150400.00 ns, 105.6301 ms/op
WorkloadActual   3: 5 op, 521447200.00 ns, 104.2894 ms/op
WorkloadActual   4: 5 op, 508700000.00 ns, 101.7400 ms/op
WorkloadActual   5: 5 op, 531727700.00 ns, 106.3455 ms/op
WorkloadActual   6: 5 op, 516529500.00 ns, 103.3059 ms/op
WorkloadActual   7: 5 op, 526874000.00 ns, 105.3748 ms/op
WorkloadActual   8: 5 op, 516808300.00 ns, 103.3617 ms/op
WorkloadActual   9: 5 op, 532089100.00 ns, 106.4178 ms/op
WorkloadActual  10: 5 op, 516883400.00 ns, 103.3767 ms/op
WorkloadActual  11: 5 op, 516735500.00 ns, 103.3471 ms/op
WorkloadActual  12: 5 op, 523296700.00 ns, 104.6593 ms/op
WorkloadActual  13: 5 op, 516306400.00 ns, 103.2613 ms/op
WorkloadActual  14: 5 op, 516488700.00 ns, 103.2977 ms/op
WorkloadActual  15: 5 op, 516507500.00 ns, 103.3015 ms/op

// AfterActualRun
WorkloadResult   1: 5 op, 514654500.00 ns, 102.9309 ms/op
WorkloadResult   2: 5 op, 528150400.00 ns, 105.6301 ms/op
WorkloadResult   3: 5 op, 521447200.00 ns, 104.2894 ms/op
WorkloadResult   4: 5 op, 508700000.00 ns, 101.7400 ms/op
WorkloadResult   5: 5 op, 531727700.00 ns, 106.3455 ms/op
WorkloadResult   6: 5 op, 516529500.00 ns, 103.3059 ms/op
WorkloadResult   7: 5 op, 526874000.00 ns, 105.3748 ms/op
WorkloadResult   8: 5 op, 516808300.00 ns, 103.3617 ms/op
WorkloadResult   9: 5 op, 532089100.00 ns, 106.4178 ms/op
WorkloadResult  10: 5 op, 516883400.00 ns, 103.3767 ms/op
WorkloadResult  11: 5 op, 516735500.00 ns, 103.3471 ms/op
WorkloadResult  12: 5 op, 523296700.00 ns, 104.6593 ms/op
WorkloadResult  13: 5 op, 516306400.00 ns, 103.2613 ms/op
WorkloadResult  14: 5 op, 516488700.00 ns, 103.2977 ms/op
WorkloadResult  15: 5 op, 516507500.00 ns, 103.3015 ms/op

// AfterAll
// Benchmark Process 5112 has exited with code 0

Mean = 104.043 ms, StdErr = 0.351 ms (0.34%), N = 15, StdDev = 1.359 ms
Min = 101.740 ms, Q1 = 103.300 ms, Median = 103.362 ms, Q3 = 105.017 ms, Max = 106.418 ms
IQR = 1.717 ms, LowerFence = 100.723 ms, UpperFence = 107.593 ms
ConfidenceInterval = [102.589 ms; 105.496 ms] (CI 99.9%), Margin = 1.453 ms (1.40% of Mean)
Skewness = 0.42, Kurtosis = 1.98, MValue = 2

// ***** BenchmarkRunner: Finish  *****

// * Export *

Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Projects\BenchmarkDotNet\samples\BenchmarkDotNet.Samples\BenchmarkDotNet.Artifacts\results\BenchmarkDotNet.Samples.IntroArgumentsSource-report.csv'.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize)
   at BenchmarkDotNet.Exporters.ExporterBase.ExportToFiles(Summary summary, ILogger consoleLogger)
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at BenchmarkDotNet.Running.BenchmarkRunnerClean.PrintSummary(ILogger logger, ImmutableConfig config, Summary summary)
   at BenchmarkDotNet.Running.BenchmarkRunnerClean.Run(BenchmarkRunInfo[] benchmarkRunInfos)
   at BenchmarkDotNet.Running.BenchmarkSwitcher.RunWithDirtyAssemblyResolveHelper(String[] args, IConfig config)
   at BenchmarkDotNet.Running.BenchmarkSwitcher.Run(String[] args, IConfig config)
   at BenchmarkDotNet.Samples.Program.Main(String[] args)
// * Artifacts cleanup *
```


```log
PS C:\Projects\BenchmarkDotNet\samples\BenchmarkDotNet.Samples> git clean -xfd        Unlink of file 'samples/BenchmarkDotNet.Samples/BenchmarkDotNet.Artifacts/BenchmarkDotNet.Samples.IntroArgumentsSource-20210114-102508.log' failed. Should I try again? (y/n) Y
Unlink of file 'samples/BenchmarkDotNet.Samples/BenchmarkDotNet.Artifacts/BenchmarkDotNet.Samples.IntroArgumentsSource-20210114-102508.log' failed. Should I try again? (y/n) n
warning: failed to remove BenchmarkDotNet.Artifacts/BenchmarkDotNet.Samples.IntroArgumentsSource-20210114-102508.log: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.dll: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.dll: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.dll: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.dll: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.dll: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/BenchmarkDotNet.Diagnostics.Windows.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/BenchmarkDotNet.Diagnostics.Windows.dll: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/BenchmarkDotNet.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/BenchmarkDotNet.dll: Invalid argument
Unlink of file 'samples/BenchmarkDotNet.Samples/bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.dll' failed. Should I try again? (y/n) n
warning: failed to remove bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.dll: Invalid argument
Removing BenchmarkDotNet.Artifacts/results
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.deps.json
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.dll.config
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.pdb
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.runtimeconfig.dev.json
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.runtimeconfig.json
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/amd64
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.pdb
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.xml
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Diagnostics.Windows.dll
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Diagnostics.Windows.pdb
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Diagnostics.Windows.xml
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.pdb
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.pdb
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/BenchmarkDotNet.xml
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/Dia2Lib.dll
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/OSExtensions.dll
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/TraceReloggerLib.dll
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/bin/Release/netcoreapp2.1/x86
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.bat
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/59fc3eed-cfb3-4860-9ce5-5cbbde129a14.notcs
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/BenchmarkDotNet.Autogenerated.csproj
Removing bin/Release/netcoreapp2.1/59fc3eed-cfb3-4860-9ce5-5cbbde129a14/obj
Removing bin/Release/netcoreapp2.1/amd64
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.pdb
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Annotations.xml
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Diagnostics.Windows.pdb
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Diagnostics.Windows.xml
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.pdb
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.deps.json
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.pdb
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.runtimeconfig.dev.json
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.Samples.runtimeconfig.json
Removing bin/Release/netcoreapp2.1/BenchmarkDotNet.xml
Removing bin/Release/netcoreapp2.1/Dia2Lib.dll
Removing bin/Release/netcoreapp2.1/OSExtensions.dll
Removing bin/Release/netcoreapp2.1/TraceReloggerLib.dll
Removing bin/Release/netcoreapp2.1/x86
Removing obj/
```

</details>

**Comment by Jason1269** [comment]
Hi @adamsitnik,

Thank you for prompt response, I am not able to reproduce with sample project I created either.  I am able to reproduce with a work project.  Will have deeper look see what edge case might have caused this and update

**Comment by hypeartist** [comment]
I have the same issue from run to run. Here is a screenshot of console output: 
![image](https://user-images.githubusercontent.com/3519388/106726673-b9114900-661b-11eb-9177-c455a6a435a6.png)

**Comment by adamsitnik** [comment]
I am closing due to lack of repro.

@hypeartist the issue you have hit should be fixed in 0.13.2 where we have merged plenty of disassembler improvements
